supernode/server/api/utils.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	"github.com/go-openapi/strfmt"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
+)
+
+// ValidateFunc validates the request parameters.
+type ValidateFunc func(registry strfmt.Registry) error
+
+// ParseJSONRequest parses the request parameter formed by JSON to a object.
+func ParseJSONRequest(body io.Reader, request interface{}, validator ValidateFunc) error {
+	if err := json.NewDecoder(body).Decode(request); err != nil {
```

Comment:
Add validation of "body == nil " and "request == nil" ?

Reply:
It should return `400` response in this case.